### PR TITLE
DOCS-2196: Versioning CC image list 

### PR DESCRIPTION
**Pull request by ctauchen**
Adds a component to create a drop-down version picker for the Calico Cloud image list. This lets users set up a private registry with older versions of CC that are still available for managed clusters.

Product Version(s):

Issue:

https://tigera.atlassian.net/browse/DOCS-2196

Link to docs preview:

https://deploy-preview-1795--tigera.netlify.app/calico-cloud/get-started/setup-private-registry#create-the-list-of-required-images
SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Dec-03-2024 17-10-25

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

File: src/___new___/components/VersionedCloudImageList/index.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    setSelectedVersion(event.target.value);
+  };
+
+  return (
+    <Flex
+      gap={4}
+      flexDir='column'
+    >
+      <Box
+        border='1px solid'
```
Comment (ronanc-tigera) [inline src/___new___/components/VersionedCloudImageList/index.js]: I would drop the border & padding. It make the select look disconnected from the code block

File: src/___new___/components/VersionedCloudImageList/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        borderColor='gray.300'
+        borderRadius='md'
+        p='4'
+        w='33%'
+      >
+        <FormControl>
+          <FormLabel>
+            <h5>Select your version:</h5>
```
Comment (ronanc-tigera) [inline src/___new___/components/VersionedCloudImageList/index.js]: You can use a chakra component for this
`<Text as='h5'>Select your version</ Text>`

File: src/___new___/components/VersionedCloudImageList/index.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+              >
+                {version}
+              </option>
+            ))}
+          </Select>
+        </FormControl>
+      </Box>
+      <CodeBlock
+        title={`Helm command to add images for Calico Cloud ${selectedVersion}`}
+        language='javascript'
```
Comment (ronanc-tigera) [inline src/___new___/components/VersionedCloudImageList/index.js:45]: this should match whatever languar is in the code block

**Review by ronanc-tigera — APPROVED** [review]

File: src/___new___/components/VersionedCloudImageList/index.js (hunk 1 above)
Comment (ctauchen) [reply src/___new___/components/VersionedCloudImageList/index.js]: done

File: src/___new___/components/VersionedCloudImageList/index.js (hunk 3 above)
Comment (ctauchen) [reply src/___new___/components/VersionedCloudImageList/index.js:45]: done

File: src/___new___/components/VersionedCloudImageList/index.js (hunk 2 above)
Comment (ctauchen) [reply src/___new___/components/VersionedCloudImageList/index.js]: Done, thanks.

**Pull request merged**